aibolit/model/model.py

```diff
@@ -151,4 +148,73 @@ def predict(self, X, quantity_func='log', th=1.0):
             except Exception:
                 raise Exception("Unknown func")
 
-        return (np.array(ranked), pairs[:, 0].T.tolist()[::-1])
+        if not return_acts:
+            return (np.array(ranked), pairs[:, 0].T.tolist()[::-1])
+        return np.array(ranked), pairs[:, 0].T.tolist()[::-1], np.zeros(X.shape[0]) - 1
+
+    def get_array(self, X, mask, i, incr):
+        """
+        Args:
+            X: np.array with shape (number of snippets, number of patterns).
+            mask: np.array with shape (number of snippets, number of patterns).
+            i: int, 0 <= i < number of patterns.
+            add: bool.
+        Returns:
+            X1: modified np.array with shape (number of snippets, ).
```

Comment:
are you sure that shape is (number of snippets, ) ?

Reply:
fix